docs/advanced.md

```diff
@@ -155,21 +155,23 @@ You can extend the K3s base template instead of copy-pasting the complete stock
   BinaryName = "/usr/bin/custom-container-runtime"
 
 ```
-## NVIDIA Container Runtime Support
 
-K3s will automatically detect and configure the NVIDIA container runtime if it is present when K3s starts.
+## NVIDIA Container Toolkit Support
+
+K3s will automatically detect and configure the NVIDIA container toolkit if it is present when K3s starts.
```

Comment:
This is not correct. We detect the runtime, not the toolkit. As per https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/index.html:
> The toolkit includes a container runtime [library](https://github.com/NVIDIA/libnvidia-container) and utilities to automatically configure containers to leverage NVIDIA GPUs.

The runtime is all that we detect, we don't actually care if you get it from the toolkit or install it standalone